mithril-aggregator/src/configuration.rs

```diff
@@ -272,21 +272,25 @@ impl Configuration {
             .map(|limit| if limit > 3 { limit as u64 } else { 3 })
     }
 
-    /// Create the deduplicated list of allowed signed entity types discriminants.
+    /// Deduce the [SignedEntityConfig] from this configuration.
+    pub fn deduce_signed_entity_config(&self) -> StdResult<SignedEntityConfig> {
```

Comment:
Maybe this is more accurate:
```suggestion
    pub fn compute_signed_entity_config(&self) -> StdResult<SignedEntityConfig> {
```